dialogue/TownNPCs/Olive/InitialInteractions/Pistachio/Pistachio_Olive_First5.json

```diff
@@ -15,11 +15,19 @@
       "Will you atleast tell me about your adventures every once in a while?",
       "That way I can make sure you're not doing anything too dangerous..!"
     ],
+    "pl": [
+      "To prawda!\nJestem taka zadowolona, że na reszcie rozumiesz.",
+      "Cz-Czekaj, co?\nSerio? Tylko wróciłaś na zmianę tempa?",
+      "W ogólę nie zmieniłaś się...",
+      "Czy najmniej możesz mi opowiedzieć o twoich przygodach raz na jakiś czas?",
```

Comment:
Use "przynajmniej" instead of "najmniej". Also "swoich" will sound more natural than "twoich", but I guess both are correct